src/components/Notification/BasicNotification.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { forwardRef, useState, useEffect } from 'react';
+import Snackbar from '@mui/material/Snackbar';
+import MuiAlert from '@mui/material/Alert';
```

Comment:
change to `import Alert from '@mui/material/Alert';`

and get rid of lines 5-7

and to the `Alert` component on line 20:
`variant="filled" elevation={6}`
will give you the same result

Reply:
This is a great change. Was there some documentation that lead you to make this change? I just grabbed the starter code from the Mui Customizable section and started editing that but I'm trying to get a better grasp on the why.

Reply:
At the bottom of each MUI component's documentation is a link to that particular API and what props it can take... the `variant` prop was there, and I think I just tried the `elevation` prop - it's not real clear but a lot of the props are "system-wide" utility ones that can be used.  Here's the `Alert` api doc: https://mui.com/material-ui/api/alert/